source/extensions/filters/listener/udpecho/BUILD

```diff
@@ -0,0 +1,35 @@
+licenses(["notice"])  # Apache 2
```

Comment:
Sorry for not being clear about this, but let's just move this into https://github.com/envoyproxy/envoy/tree/master/test/integration/filters and then we can void documentation, extension config, etc.

Reply:
Sure.